### DISCUSSION TITLE
Security Questionnaire risk assessment - Supplier Evaluation SOC2

### DISCUSSION THREAD

**Discussion by laurasuarezrpo** [community/community#120111] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello hello! 

We are currently undergoing a comprehensive risk assessment in our organization to obtain our SOC2 certification. As part of this assessment, it is essential that we collect detailed security information from all our key suppliers.

Given the importance of the GitHub service to our operations, we would like to include you in this process, so we would like to know at what email address or via what medium you (GitHub) can receive our vendor security questionnaire.

We greatly appreciate your cooperation and support in this critical matter for compliance and continuous improvement of our safety practices.

**Comment by ebndev** [comment]
Hi @laurasuarezrpo,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
